### PR TITLE
Joi

### PR DESCRIPTION
**Pull request by Danwhy**
ref #110 
Removes validation from the config and does it manually on the parameters and queries as they are received in the request. This is because the jsonapi request and the html request accept different data types

### CODE REVIEW

**Comment by nelsonic** [comment]
https://youtu.be/GVUTiuPuxMc?t=6s

**Comment by SimonLab** [comment]
Good for me!

File: test/search-filters.test.js [hunk 1]
```diff
@@ -739,3 +889,18 @@ testWithServer('Should accept multiple image_licences', (t, ctx) => {
     t.end();
   });
 });
+
+testWithServer('Should not accept array of multiple image_licences as json', (t, ctx) => {
+  t.plan(1);
```
Comment (nelsonic) [inline test/search-filters.test.js]: do we need _both_ `t.plan(1);` _and_ `t.end();` ?

Comment (Danwhy) [inline test/search-filters.test.js]: Possibly not, but [Alan suggested that we should](https://github.com/TheScienceMuseum/collectionsonline/pull/28#issuecomment-226163479)

Comment (nelsonic) [inline test/search-filters.test.js]: Ok. cool.

**Pull request merged**